### PR TITLE
OpenAsync method is blocked

### PR DESCRIPTION
**Pull request by AigizK**

### CODE REVIEW

File: FoundationDB.Client/Fdb.cs [hunk 1]
```diff
@@ -526,7 +526,7 @@ public static Task<IFdbDatabase> OpenAsync(string clusterFile, string dbName, Ca
 		[ItemNotNull]
 		public static async Task<IFdbDatabase> OpenAsync(string clusterFile, string dbName, IKeySubspace globalSpace, bool readOnly = false, CancellationToken ct = default)
 		{
-			return await OpenInternalAsync(clusterFile, dbName, globalSpace, readOnly, ct);
+			return await OpenInternalAsync(clusterFile, dbName, globalSpace, readOnly, ct).ConfigureAwait(false);
```
Comment (KrzysFR) [inline FoundationDB.Client/Fdb.cs]: I think this method should not be async anyway, it only return a task. We should get rid of the async/away + ConfigureAwait(false) altogether.

Comment (KrzysFR) [reply FoundationDB.Client/Fdb.cs]: Oh I forgot that OpenInternalAsync returns a Task<FdbDatabase>, so the await is required to cast the result back into IFdbDatabase. This should be changed.

**Comment by KrzysFR** [comment]
I have fixed this in 9de529b2122f1eb6d9ccd9de95bb5ca565cc70d9, which removes one extra async state machine. Let me know if this fixes your problem

**Comment by AigizK** [comment]
yes, thank you

**Pull request closed without merging**